R/bootstrap-layout.R

```diff
@@ -243,6 +247,7 @@ column <- function(width, ..., offset = 0) {
 #'   `title` tag within the head. You can also specify a page title
 #'   explicitly using the `title` parameter of the top-level page function.
 #'
+#' @family layout functions
 #'
```

Comment:
I don't think `titlePanel()` should be included in the layout family. It just provides a way to specify the title of the page.